### DISCUSSION TITLE
SATA 0 + Port Multiplier

### DISCUSSION THREAD

**Discussion by demlak** [R3NE07/Futro-S740#24] [Ideas]
Moin..
ich habe Interesse an mehr SATA Ports.. da ich aber bereits Geld für eine `WD Blue SA510, M.2 2280, 500GB, SATA III` ausgegeben habe, fällt wohl der `Chenyang NGFF Schlüssel B + M zu SATA 3,0 6Gbps 5 Ports Adapter Konverter Port Multiplier Selector JMB575 2280` flach..

Nun ist es ja [relativ einfach einen SATA0 Port hinzuzufügen](https://www.mydealz.de/comments/permalink/38114275).. da frage ich mich gerade, ob man einen SATA Multiplexer [wie diesen hier](https://de.aliexpress.com/item/1005005139475403.html) nutzen kann. Kann da jemand was zu sagen?

**Comment by Viper780** [comment]
Warum verwendest du nicht den anderen m.2 Slot, der für WiFi.

**Reply by demlak** [reply]
Ich hab mehrere Variationen im Hinterkopf um auf mehrere SATA-Ports zu kommen.. Welche genau meinst du?

**Reply by Viper780** [reply]
Wieviele Ports brauchst du?

Ich hab sowas in Verwendung https://www.amazon.de/Express-Adapter-Converter-Festplattenerweiterungskarte-JMB582/dp/B09MTJQKB5/

Gibt's mit beliebig vielen SATA Ports.
Bei den größeren müsste man mit M.2 Risern arbeiten

**Reply by demlak** [reply]
Der Adapter ist mir bekannt.. ich brauche aber 4 Ports..

Was heißt mit beliebig vielen Ports? für den A+E-Key Slot ist mir nur dieser mit zwei Ports bekannt.

Der 5-Port Adapter, den ich oben selber genannt habe, passt ja nicht in den Slot
